lonestar/analytics/distributed/README.md

```diff
@@ -54,15 +54,15 @@ command line option:
 `./bfs_push -help`
 
 Most of the provided graph applications take graphs in a .gr format, which
-is a Galois graph format that stores the graph in a CSR or CSC format. We 
+is a Galois graph format that stores the graph in a CSR or CSC format. We
 provide a graph converter tool under 'tools/graph-convert' that can take
 various graph formats and convert them to the Galois format.
 
 Running Provided Apps (Distributed Apps)
 ================================================================================
 
 First, note that if running multiple processes on a single machine, specifying
```

Comment:
It is not clear when multiple processes will be run. You can modify it to "machine (e.g., running on a single-host multi-GPU or multi-host multi-GPU machine)